examples/model_selection/grid_search_text_feature_extraction.py

```diff
@@ -88,8 +90,11 @@
     ]
 )
 
-# uncommenting more parameters will give better exploring power but will
-# increase processing time in a combinatorial way
+# %%
+# Grid Search
+# -----------
```

Comment:
Maybe splitting this as a new cell is not really necessary if my previous comment is taken into account, what do you think?

Reply:
Yeah I agree. I kept it because it was there previously, but I don't think it is needed